### ISSUE TITLE
[BUG] FAILED forecasting/tests/test_ets.py::test_airline_allow_multiplicative_trend on linux and py39

### ISSUE THREAD

**Issue by lmmentel** [sktime/sktime#1747]
**Describe the bug**

Test is failing:

```
[gw0] linux -- Python 3.9.9 /opt/hostedtoolcache/Python/3.9.9/x64/bin/python

    def test_airline_allow_multiplicative_trend():
        fit_result_R = ["mul", "mul", "mul"]
    
        forecaster = AutoETS(auto=True, sp=12, n_jobs=-1, allow_multiplicative_trend=True)
        forecaster.fit(y)
        fitted_forecaster = forecaster._fitted_forecaster
        fit_result = [
            fitted_forecaster.error,
            fitted_forecaster.trend,
            fitted_forecaster.seasonal,
        ]
    
>       assert_array_equal(fit_result_R, fit_result)
E       AssertionError: 
E       Arrays are not equal
E       
E       Mismatched elements: 1 / 3 (33.3%)
E        x: array(['mul', 'mul', 'mul'], dtype='<U3')
E        y: array(['mul', 'add', 'mul'], dtype='<U3')

fit_result = ['mul', 'add', 'mul']
fit_result_R = ['mul', 'mul', 'mul']
fitted_forecaster = <statsmodels.tsa.exponential_smoothing.ets.ETSResultsWrapper object at 0x7fb404124040>
forecaster = AutoETS(additive_only=False, allow_multiplicative_trend=True, auto=True,
        bounds=None, callback=None, damped_tr...ing='none', n_jobs=-1, restrict=True, return_params=False,
        seasonal=None, sp=12, start_params=None, trend=None)

/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/site-packages/sktime/forecasting/tests/test_ets.py:62: AssertionError
```

**To Reproduce**

Run the test on linux with python 3.9, see https://github.com/lmmentel/sktime/runs/4498946774?check_suite_focus=true

**Expected behavior**

Test passes.

**Additional context**

**Versions**

OS: linux ubuntu 20.04
python: 3.9.9

```
Run python -m pip list
Package                           Version
--------------------------------- ---------
attrs                             21.2.0
backports.entry-points-selectable 1.1.1
certifi                           2021.10.8
cfgv                              3.3.1
charset-normalizer                2.0.9
click                             8.0.3
cloudpickle                       2.0.0
cmdstanpy                         0.9.5
convertdate                       2.3.2
coverage                          6.2
cycler                            0.11.0
Cython                            0.29.25
dask                              2021.12.0
Deprecated                        1.2.13
distlib                           0.3.4
distributed                       2021.12.0
dtw-python                        1.1.10
ephem                             4.1.2
esig                              0.9.7
execnet                           1.9.0
fbprophet                         0.7.1
filelock                          3.4.0
fonttools                         4.28.3
fsspec                            2021.11.1
hcrystalball                      0.1.10
HeapDict                          1.0.1
hijri-converter                   2.2.2
holidays                          0.11.3.1
identify                          2.4.0
idna                              3.3
iniconfig                         1.1.1
Jinja2                            3.0.3
joblib                            1.1.0
kiwisolver                        1.3.2
korean-lunar-calendar             0.2.1
llvmlite                          0.37.0
locket                            0.2.1
LunarCalendar                     0.0.9
lunardate                         0.2.0
MarkupSafe                        2.0.1
matplotlib                        3.5.1
matrixprofile                     1.1.10
msgpack                           1.0.3
nodeenv                           1.6.0
numba                             0.54.1
numpy                             1.19.3
packaging                         21.3
pandas                            1.3.5
partd                             1.2.0
patsy                             0.5.2
Pillow                            8.4.0
pip                               21.3.1
platformdirs                      2.4.0
pluggy                            1.0.0
pmdarima                          1.8.4
pre-commit                        2.16.0
protobuf                          3.11.2
psutil                            5.8.0
py                                1.11.0
pyluach                           1.3.0
PyMeeus                           0.5.11
pyod                              0.9.5
pyparsing                         3.0.6
pystan                            2.19.1.1
pytest                            6.2.5
pytest-cov                        3.0.0
pytest-forked                     1.4.0
pytest-xdist                      2.5.0
python-dateutil                   2.8.2
pytz                              2021.3
PyYAML                            6.0
requests                          2.26.0
scikit-learn                      1.0.1
scikit-posthocs                   0.6.7
scipy                             1.7.3
seaborn                           0.11.2
setuptools                        58.1.0
setuptools-git                    1.2
six                               1.16.0
sktime                            0.8.1
sortedcontainers                  2.4.0
statsmodels                       0.12.1
stumpy                            1.10.0
tbats                             1.1.0
tblib                             1.7.0
threadpoolctl                     3.0.0
toml                              0.10.2
tomli                             1.2.2
toolz                             0.11.2
tornado                           6.1
tqdm                              4.62.3
tsfresh                           0.18.0
urllib3                           1.26.7
virtualenv                        20.10.0
wheel                             0.37.0
workalendar                       16.2.0
wrapt                             1.13.3
zict                              2.0.0
```

</details>

**Comment by lmmentel** [comment]
@HYang1996 I saw some of your comments in the [tests source](https://github.com/alan-turing-institute/sktime/blob/fa2d381e3f58f3ddc1e277e2202086715c4d8b5b/sktime/forecasting/tests/test_ets.py#L40) do you remember what was the issue or how you solved that?

**Comment by lmmentel** [comment]
It seems that despite `allow_multiplicative_trend=True` parameter the estimator return `"add"` for the trend component. Weird that this only happens for python 3.9 on linux and not any other python-os combination. Any thoughts?

**Comment by fkiraly** [comment]
well, "allow" means that `"mul"` *can* happen, not that it *must* happen as result of fit.
I do not understand why it would be different for different python versions, perhaps some random seed behaving differently?

Generally, I think tests like this which test fitted parameters against expected fitted parameters make sense but can be brittle. I suggest to simply mark it with an `xfail` and keep it open as a bug report. It's probably coming from the modelling package dependency, so we may want to localize this bug and post there once we can isolate it to the dependency.

**Comment by lmmentel** [comment]
This should be resolved by 03f0b96acde1cd0fe804c0b071f4fa3c2456cab3 in #1620 with `xfail`.
